bgpd/bgp_vty.c

```diff
@@ -16867,8 +16867,6 @@ void bgp_vty_init(void)
 	install_element(BGP_VPNV6_NODE, &no_neighbor_unsuppress_map_cmd);
 
 	/* neighbor maximum-prefix-out commands. */
-	install_element(BGP_NODE, &neighbor_maximum_prefix_out_cmd);
-	install_element(BGP_NODE, &no_neighbor_maximum_prefix_out_cmd);
```

Comment:
I'm not sure but we might need to do a deprecation cycle before removing those commands.

Reply:
It was added to 7.4 only, actually.